docs/dashboards/detection-response-dashboard.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[detection-response-dashboard]]
+= Detection & Response dashboard
+
+The Detection & Response dashboard provides focused visibility into the day-to-day operational activities of your security environment. Security operations managers and analysts can quickly monitor recent and high priority detection alerts and cases, and identify the hosts and users associated with frequent and severe alerts.
```

Comment:
```suggestion
The Detection & Response dashboard provides focused visibility into the day-to-day operations of your security environment. It helps security operations managers and analysts quickly monitor recent and high priority detection alerts and cases, and identify the hosts and users associated with alerts.
```
Re: the deletion I suggested in the last line, even though it does highlight the most frequent and severe alerts if there are any, since it enables monitoring of all alerts I think this would be more accurate.